src/scraper/scraper.cpp

```diff
@@ -1573,17 +1710,43 @@ bool DownloadProjectRacFilesByCPID(const NN::WhitelistSnapshot& projectWhitelist
                 _log(logattribute::CRITICAL, "DownloadProjectRacFiles", "Failed to export etag for " + prjs.m_name + " to authentication file");
         }
 
-        std::string chketagfile = prjs.m_name + "-" + sRacETag + ".csv" + ".gz";
-        fs::path chkfile = pathScraper / chketagfile;
+        std::string rac_file_name;
+        fs::path rac_file;
+
+        std::string processed_rac_file_name;
+        fs::path processed_rac_file;
 
-        if (fs::exists(chkfile))
+        processed_rac_file_name = prjs.m_name + "-" + sRacETag + ".csv" + ".gz";
+        processed_rac_file = pathScraper / processed_rac_file_name;
+
+        if (fExplorer)
         {
-            _log(logattribute::INFO, "DownloadProjectRacFiles", "Etag file for " + prjs.m_name + " already exists");
-            //_log(logattribute::INFO, "DownloadProjectRacFiles", "Etag file size " + std::to_string(fs::file_size(chkfile)));
-            continue;
+            // Use eTag versioning for source file.
+            rac_file_name = prjs.m_name + sRacETag + "-user.gz";
```

Comment:
Minor: I noticed that the _user_ unprocessed stats files don't have a hyphen between the project name and etag...might be good for those names to match.

Reply:
Dammit. That pisses me off that I missed that. Now I have an f*d up directory for the test node I have been running, and I am not going to edit every one of those files. Time to blow the scraper directory away.

Reply:
I guess I can just let the old ones flush out. They are not processed, so it doesn't matter if they aren't named right.

Reply:
I think my scraper directory is up to >30 GB, because it has one week stored for most (not all of the user files)...

Reply:
Good thing only startail is going to be using this most likely...

Reply:
Heh, yup... I won't be able to in prod... my VPS has a ~50 GB disk I think.